### PR TITLE
Fix failing cucumber test + unit test + remove unused code

### PR DESCRIPTION
**Pull request by thomshutt**

### CODE REVIEW

File: transcode/transcode.go [hunk 1]
```diff
@@ -76,18 +76,6 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
...
-	targetTranscodedOutputPath := path.Join(targetTranscodedPath, targetTranscodedManifestFilename)
-	// Generate the manifest output URL from the manifest output path (e.g. s3+https://USER:PASS@storage.googleapis.com/user/hls/index.m3u8)
-	tpath, err := url.Parse(targetTranscodedOutputPath)
-	if err != nil {
-		return outputs, fmt.Errorf("failed to parse targetTranscodedOutputPath: %s", err)
-	}
-	targetTranscodedOutputURL := segmentedOutputManifestURL.ResolveReference(tpath)
-	fmt.Println(targetTranscodedOutputURL)
-	// Generate the rendition output URL (e.g. s3+https://USER:PASS@storage.googleapis.com/user/hls/)
 	tout, err := url.Parse(targetTranscodedPath)
```
Comment (emranemran) [inline transcode/transcode.go:79]: Don't we need the above deleted code to generate the filenames?

Comment (thomshutt) [reply transcode/transcode.go:79]: targetTranscodedOutputURL is just being printed though and not used, which is why I could delete it

Comment (emranemran) [reply transcode/transcode.go:79]: ooh yeah i left that there for a follow up PR to change the rendition folder structures. I'll add it back when I put up that PR.

**Review by emranemran — APPROVED** [review]

**Review by red-0ne — APPROVED** [review]
LGTM, had to resolve cucumber tests and `TestItCanTranscode` almost the same way on my current branch.

**Pull request merged**